app/pages/Import/FileUpload/index.tsx

```diff
@@ -16,7 +16,7 @@ const FileUpload = () => {
   const [loading, setLoading] = useState(false);
   const transformFile = async (_file: StudioFile, fileList: StudioFile[]) => {
     const size = fileList.reduce((acc, cur) => acc + cur.size, 0);
-    if(size > global.gConfig.maxBytes) {
+    if(global.gConfig.maxBytes && size > global.gConfig.maxBytes) {
```

Comment:
what is `maxBytes` default value if config item is empty ?

Reply:
I think there is no need to execute this logic if it is not set or it is not read for some reason, it is better to report 413 directly

Reply:
there is default value for `maxBytes` config, not setting doesn't mean it not work, see the doc below and make tests
https://go-zero.dev/docs/configuration/api?_highlight=maxbytes#restrestconf

Reply:
go-zero@1.3.3
```go
RestConf struct {
	service.ServiceConf
	Host     string `json:",default=0.0.0.0"`
	Port     int
	CertFile string `json:",optional"`
	KeyFile  string `json:",optional"`
	Verbose  bool   `json:",optional"`
	MaxConns int    `json:",default=10000"`
	MaxBytes int64  `json:",default=1048576"`
	// milliseconds
	Timeout      int64         `json:",default=3000"`
	CpuThreshold int64         `json:",default=900,range=[0:1000]"`
	Signature    SignatureConf `json:",optional"`
}
```

Reply:
Can't read the default value